### ISSUE TITLE
Remove GraphQL Support

### ISSUE THREAD

**Issue by engelgabriel** [RocketChat/Rocket.Chat#14959]
The current implementation of our GraphQL API was an experiment that for many reasons did not get much adoption. It runs on a separate port and we were never able to fully integrate into the main code. For that reason, it has added too many dependencies that are useless for the vast majority of our deployments. It is the source of annoying issues like https://github.com/RocketChat/Rocket.Chat/issues/10714 causing unnecessary load and worries. 

We believe that supporting GraphQL is still on our long term plans, but we will have to refactor the implementation with the new architecture we are designing for high scalability.

With that in mind, we are **planning to remove the current implementation** from the code base and keep the PR https://github.com/RocketChat/Rocket.Chat/pull/8158 as a reference for future development as an auxiliary Apolo base project (_probably_).

**Comment by winterstefan** [comment]
Hey guys,

since I was send over from one of my [pull requests](https://github.com/RocketChat/Rocket.Chat/issues/13546#issuecomment-510619759), here're my five cents. :-)

**tl;dr**
We're not using GraphQL API. It would be okay for me, if you remove the support for it (although it would be cool having a brand new neat solution).

**detailed** 
With my company, we've created a custom implementation of Rocket.Chat with our website. For that, we're using both the REST API and the Realtime API via ddp.
In fact, we'd have a bunch of possible features & improvements where a generic GraphQL API would be appreciated.

For Example:
_Displaying incoming messages to a certain channel (which is connected to an entity on the website)._

To achieve that even for anonymous users (or at least those who're not part of the channel), we experimented with the Rocket.Chat GraphQL API.
But that's not production-ready, so *removing your GraphQL implementation wouldn't affect us*.
